SimGeneral/MixingModule/python/castorDigitizer_cfi.py

```diff
@@ -12,9 +12,10 @@
         samplingFactor = cms.double(0.062577), ## GeV/pe
 
         doPhotoStatistics = cms.bool(True),
-        photoelectronsToAnalog = cms.double(4.009),
+        photoelectronsToAnalog = cms.double(4.1718),
```

Comment:
@hvanhaev @civanch just to be sure, is this change not expected to modify anyway the result, regardless of the noise treatment?

Reply:
This is also regulated by the parameter doDynamicNoise, which when false preserves the old behaviour, see
https://github.com/cms-sw/cmssw/pull/23894/files#diff-ddf1df6e7e05d46bf1348290efb42847R48